### DISCUSSION TITLE
Feature for printing errors

### DISCUSSION THREAD

**Discussion by sameertri039** [cpeditor/cpeditor#1123] [Ideas]
Hi, 
Just used the editor and found it to be very good.
I would like the developers to add a new error window just like the input and output window for displaying the errors when using cerr instead of cout in C++. Maybe add an option for such things in the preference settings.

I use a debugging template and when using cerr for debugging my codes, I can't get the output anywhere. I have to use cout which sometimes wrong answer while submitting.
Thanks

**Comment by ouuan** [comment]
It should be displayed in the "Messages" window:

![screenshot](https://user-images.githubusercontent.com/30581822/205457280-86a3c6b0-31b4-4688-a2e3-02b041ad0d72.png)
